src/test/resources/application-test.yml

```
@@ -15,17 +21,19 @@ spring:
-  datasource:
-    url: jdbc:mysql://localhost:3306/dummy
-    username: dummy
-    password: dummy
-    driver-class-name: com.mysql.cj.jdbc.Driver
 
-  data:
-    redis:
-      host: dummy
-      port: 6379
-      password:
+
```

YAMLlint (1.37.1)

[error] too many blank lines (3 > 2)

(empty-lines)